action/action.go

```diff
@@ -24,6 +24,11 @@ const (
 	BlobTxType       = 4
 )
 
+var (
+	// block height = 18879571, tx d03127451c120caed0c4e2ed91c58872231192e027daad3760c475e5aa2feffd used this value
+	_outOfBandTxType18879571 uint32 = 4689
```

Comment:
obviously, this is the evmNetworkID.

Reply:
yes, but it is a co-incidence. someone just send in a tx at height 18879571 by setting `version = 4689`, that's why name it that way